src/Haskell/Debug/Adapter/State/DebugRun.hs

```diff
@@ -222,6 +223,14 @@ instance StateActivityIF DebugRunStateData DAP.SetFunctionBreakpointsRequest whe
     liftIO $ L.debugM _LOG_APP $ "DebugRunState SetFunctionBreakpointsRequest called. " ++ show req
     SU.setFunctionBreakpointsRequest req
 
+-- |
+--  Any errors should be sent back as False result Response
+--
+instance StateActivityIF DebugRunStateData DAP.SourceRequest where
```

Comment:
Could you move this implementation (including Utility.hs code together)
to Debug/Adapter/State/DebugRun/Source module ?

Reply:
Thank you for your comment!
I moved them. https://github.com/phoityne/haskell-debug-adapter/pull/27/commits/88867ead45c7c7bffdc4d4f1b26bf40b441acd50